docs/roadmap.md

```diff
@@ -0,0 +1,35 @@
+# Roadmap
+
+This is the official roadmap for the Open Service Broker for Azure (OSBA) project. OSBA is currently available as an alpha release and this roadmap represents our plan to deliver (OSBA) and the services it provides in a stable form.
+
+This roadmap addresses stability in terms of two dimensions:
+
+* Services/Plans
+* Database Schemas
+
+See below for more information concerning stability plans for each.
+
+# Services and Plans
+
+Services/Plans have three stability tiers:
+
+- `experimental` - We have an idea for a new service that we want to support, but don't have a good idea how it should look. We are essentially “throwing something against the wall” to see if it'll stick. Experimental services may be radically changed or removed at any time
+- `preview` - We have a better understanding how a service should look, but we don't yet guarantee backward compatibility. We do guarantee that this service won't go back to `experimental`, so we are committing to making it exist in some shape or form
+- `stable` - We now understand usage patterns of the service very well and we guarantee full backward compatibility
+
+Here are our timelines for service stabilities:
+
+- SQL Server: `preview` by the end of February 2018
+- MySQL: `preview` by the end of February 2018
+- PostgreSQL: `preview` by the end of February 2018
+
+We aren't yet setting timelines on other service stabilities.
+
+# Database Schemas
+
+There are two tiers of database schemas to consider:
```

Comment:
Can you explain here why database schemas are important to consider for stability?